docs/admin/administration.md

```diff
@@ -606,36 +613,89 @@ an evicted device.
 
 - Locate a Healthy SSD:
 ```bash
-$ dmg storage identify vmd --help
+$ dmg storage led identify --help
 Usage:
-  dmg [OPTIONS] storage identify vmd [vmd-OPTIONS]
+  dmg [OPTIONS] storage led identify [identify-OPTIONS] [ids]
 
 ...
 
-[vmd command options]
-          --uuid=     Device UUID of the VMD device to identify
+[identify command options]
+          --reset     Reset blinking LED on specified VMD device back to previous state
+
+[identify command arguments]
+  ids:                Comma-separated list of identifiers which could be either VMD backing device
+                      (NVMe SSD) PCI addresses or device
 ```
 
-To quickly identify an SSD in question, an administrator can run the following
-command:
+To identify a single SSDs, any of the Device-UUIDs can be used which can be found from
```

Comment:
minor typo: SSDs -> SSD

Reply:
will correct in follow-on PR

Reply:
addressing in https://github.com/daos-stack/daos/pull/10768